protostar/protostar_toml/io/protostar_toml_reader.py

```diff
@@ -46,15 +45,31 @@ def get_attribute(
         if not section:
             return None
 
-        if attribute_name not in section:
-            attribute_name = attribute_name.replace("-", "_")
-            if attribute_name not in section:
-                return None
+        alternative_attribute_name = self._find_alternative_key(attribute_name, section)
+
+        if alternative_attribute_name and alternative_attribute_name in section:
+            return section[alternative_attribute_name]
+        return None
+
+    # pylint: disable=no-self-use
+    def _find_alternative_key(
```

Comment:
Why  pylint ignore? staticmethod would be better

Reply:
Static methods extend the interface of this class. I rather keep interface smaller than do a micro-optimization. 

Making it static or not doesn't matter.

Reply:
using static is not matter of optimization idk. Is a guarantee that method does not alter object state, which is a VERY important information for developer.

Reply:
This information comes with the cost of encouraging bad patterns.

I need to log the warning anyway, so this method can't be static anymore.

Reply:
> This information comes with the cost of encouraging bad patterns.
I don't see any bad patterns here, that is a purpose why staticmethod exists.